migrate/repo_checks.py

```diff
@@ -500,7 +500,6 @@ def _update_branch_protection(self, params):
             # If the branch is missing, the repo is probably new and has no branches
             # ignore that for now.
             if resp.json.get("message") != "Branch not found":
```

Comment:
```suggestion
            if resp.json().get("message") != "Branch not found":
```
I think this is also a typo.